### ISSUE TITLE
[Modular CLI]: Installation of Mojo fails (fedora-39)

### ISSUE THREAD

**Issue by allemand-instable** [modularml/mojo#1934]
## issue encoutentered

- mojo installation failed 
- seems to be missing libraries ( might be because of the distribution : fedora 39 )
    -  any method to get it working as expected ?

## expected behaviour

- Mojo installation

## output lines that caught my attention

```
[...]
/lib64/libtinfo.so.6: no version information available
[...]
error while loading shared libraries: libedit.so.2: cannot open shared object file: No such file or directory
[...]
```

## full console print since the command

```
❯ modular install mojo

modular: /lib64/libtinfo.so.6: no version information available (required by modular)
# Found release for https://packages.modular.com/mojo @ 24.1.0-55ec12d6-release
# Downloading archive: packages/24.1.0-55ec12d6-release/mojo-x86_64-unknown-linux-gnu-24.1.0-55ec12d6-release.tar.gz

Downloading  [ ███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████ ] 100%     131MiB/131MiB @  25.3MiB/s

# Running pre-uninstall hooks for the old version...
# Extracting downloaded archives. 
# Extraction complete, setting configs...
# Configs complete, running post-install hooks...

Requirement already satisfied: find_libpython==0.3.0 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from -r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 1)) (0.3.0)
Requirement already satisfied: jupyter_client>=8.3.0 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from -r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2)) (8.4.0)
Requirement already satisfied: ipykernel>=6.25.0 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from -r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (6.25.2)
Requirement already satisfied: tornado>=6.2 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from jupyter_client>=8.3.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2)) (6.3.3)
Requirement already satisfied: traitlets>=5.3 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from jupyter_client>=8.3.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2)) (5.11.2)
Requirement already satisfied: pyzmq>=23.0 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from jupyter_client>=8.3.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2)) (25.1.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from jupyter_client>=8.3.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2)) (2.8.2)
Requirement already satisfied: jupyter-core!=5.0.*,>=4.12 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from jupyter_client>=8.3.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2)) (5.4.0)
Requirement already satisfied: nest-asyncio in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (1.5.8)
Requirement already satisfied: debugpy>=1.6.5 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (1.8.0)
Requirement already satisfied: comm>=0.1.1 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (0.1.4)
Requirement already satisfied: packaging in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (23.2)
Requirement already satisfied: psutil in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (5.9.6)
Requirement already satisfied: ipython>=7.23.1 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (8.16.1)
Requirement already satisfied: matplotlib-inline>=0.1 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (0.1.6)
Requirement already satisfied: exceptiongroup in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (1.1.3)
Requirement already satisfied: pexpect>4.3 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (4.8.0)
Requirement already satisfied: jedi>=0.16 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (0.19.1)
Requirement already satisfied: pickleshare in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (0.7.5)
Requirement already satisfied: decorator in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (5.1.1)
Requirement already satisfied: stack-data in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (0.6.3)
Requirement already satisfied: pygments>=2.4.0 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (2.16.1)
Requirement already satisfied: backcall in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (0.2.0)
Requirement already satisfied: prompt-toolkit!=3.0.37,<3.1.0,>=3.0.30 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (3.0.39)
Requirement already satisfied: platformdirs>=2.5 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from jupyter-core!=5.0.*,>=4.12->jupyter_client>=8.3.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2)) (3.11.0)
Requirement already satisfied: six>=1.5 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from python-dateutil>=2.8.2->jupyter_client>=8.3.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 2)) (1.16.0)
Requirement already satisfied: parso<0.9.0,>=0.8.3 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from jedi>=0.16->ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (0.8.3)
Requirement already satisfied: ptyprocess>=0.5 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from pexpect>4.3->ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (0.7.0)
Requirement already satisfied: wcwidth in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from prompt-toolkit!=3.0.37,<3.1.0,>=3.0.30->ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (0.2.8)
Requirement already satisfied: asttokens>=2.1.0 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from stack-data->ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (2.4.0)
Requirement already satisfied: pure-eval in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from stack-data->ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (0.2.2)
Requirement already satisfied: executing>=1.2.0 in /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/python3.10/site-packages (from stack-data->ipython>=7.23.1->ipykernel>=6.25.0->-r /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/requirements.txt (line 3)) (2.0.0)


[notice] A new release of pip is available: 23.0.1 -> 24.0

[notice] To update, run: pip install --upgrade pip

[mojo][error] Mojo SDK post-install test "`mojo repl`" failed:
SubprocessError: Command '/home/allemandinstable/.modular/pkg/packages.modular.com_mojo/bin/mojo repl --one-line-before-file settings set target.load-cwd-lldbinit false --one-line-before-file settings set plugin.process.gdb-remote.packet-timeout 60' returned non-zero exit status 127.
  stderr:
    /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/bin/mojo: /lib64/libtinfo.so.6: no version information available (required by /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/bin/mojo)
    /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/bin/mojo-lldb: error while loading shared libraries: libedit.so.2: cannot open shared object file: No such file or directory
  File "/home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/self-test.py", line 303, in test_mojo_repl
    raise SubprocessError(repl.returncode, " ".join(args), out, err)
[mojo][error] Some components of the Mojo SDK may have been installed successfully, but others may not work as expected.
[mojo][error] Please submit an issue to https://github.com/modularml/mojo and include the full output of the command you just ran.


modular: error: failed to run script

==========================
Failure Information:
	- Script: `PATH=/home/allemandinstable/.local/share/pnpm:/home/allemandinstable/.modular/pkg/packages.modular.com_mojo/bin:/home/allemandinstable/.pyenv/shims:/home/allemandinstable/.pyenv/bin:/usr/local/cuda-12.1/bin:/home/allemandinstable/.modular/pkg/packages.modular.com_mojo/bin:/home/allemandinstable/.local/bin:/home/allemandinstable/bin:/opt/devkitpro/tools/bin:/opt/devkitpro/tools/bin:/usr/lib64/ccache:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/var/lib/snapd/snap/bin:/home/allemandinstable/go_path/bin HOME=/home/allemandinstable MODULAR_HOME=/home/allemandinstable/.modular PYTHONPATH=/home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/lib/home/allemandinstable/.modular/pkg/packages.modular.com_mojo/venv/bin/python /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/self-test.py --modular-home /home/allemandinstable/.modular --package-path /home/allemandinstable/.modular/pkg/packages.modular.com_mojo --mojo-config-section mojo`
	- Result: 1
	- Stderr:
[mojo][error] Mojo SDK post-install test "`mojo repl`" failed:
SubprocessError: Command '/home/allemandinstable/.modular/pkg/packages.modular.com_mojo/bin/mojo repl --one-line-before-file settings set target.load-cwd-lldbinit false --one-line-before-file settings set plugin.process.gdb-remote.packet-timeout 60' returned non-zero exit status 127.
  stderr:
    /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/bin/mojo: /lib64/libtinfo.so.6: no version information available (required by /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/bin/mojo)
    /home/allemandinstable/.modular/pkg/packages.modular.com_mojo/bin/mojo-lldb: error while loading shared libraries: libedit.so.2: cannot open shared object file: No such file or directory
  File "/home/allemandinstable/.modular/pkg/packages.modular.com_mojo/scripts/post-install/self-test.py", line 303, in test_mojo_repl
    raise SubprocessError(repl.returncode, " ".join(args), out, err)
[mojo][error] Some components of the Mojo SDK may have been installed successfully, but others may not work as expected.
[mojo][error] Please submit an issue to https://github.com/modularml/mojo and include the full output of the command you just ran.
```

## command used :

```
modular install mojo
```

## error that appears on every single command :

```
modular: /lib64/libtinfo.so.6: no version information available (required by modular)
```

### Version Info

```shell
## modular version


❯ modular -v

modular: /lib64/libtinfo.so.6: no version information available (required by modular)

modular 0.5.1 (1b608e3d)
```

## system 

- Fedora 39

## Python installation :

```
❯ python
Python 3.12.0 (main, Oct 30 2023, 12:17:12) [GCC 13.2.1 20231011 (Red Hat 13.2.1-4)] on linux
```

```
❯ pyenv virtualenvs
  3.11.5/envs/manim (created from /home/allemandinstable/.pyenv/versions/3.11.5)
  3.11.5/envs/ML (created from /home/allemandinstable/.pyenv/versions/3.11.5)
  3.12.0/envs/base (created from /home/allemandinstable/.pyenv/versions/3.12.0)
* base (created from /home/allemandinstable/.pyenv/versions/3.12.0)
  manim (created from /home/allemandinstable/.pyenv/versions/3.11.5)
  ML (created from /home/allemandinstable/.pyenv/versions/3.11.5)

❯ pyenv global
system

❯ dnf list installed | rg python
[...]
python2.7.x86_64                                     2.7.18-37.fc39                      @updates                                                       
python3.x86_64                                       3.12.2-2.fc39                         @updates          
[...]
```

## dev libraries installed on the fedora system :

```
❯ dnf list installed | rg devel
R-core-devel.x86_64                                  4.3.2-1.fc39                        @updates                                                       
R-devel.x86_64                                       4.3.2-1.fc39                        @updates                                                       
R-java-devel.x86_64                                  4.3.2-1.fc39                        @updates                                                       
brotli-devel.x86_64                                  1.1.0-1.fc39                        @fedora-modular                                                
bzip2-devel.x86_64                                   1.0.8-16.fc39                       @fedora-modular                                                
cairo-devel.x86_64                                   1.18.0-1.fc39                       @fedora-modular                                                
elfutils-debuginfod-client-devel.x86_64              0.190-4.fc39                        @updates                                                       
elfutils-devel.x86_64                                0.190-4.fc39                        @updates                                                       
elfutils-libelf-devel.x86_64                         0.190-4.fc39                        @updates                                                       
flexiblas-devel.x86_64                               3.4.1-1.fc39                        @updates                                                       
fontconfig-devel.x86_64                              2.14.2-6.fc39                       @updates                                                       
freeglut-devel.x86_64                                3.4.0-7.fc39                        @updates                                                       
freetype-devel.x86_64                                2.13.1-2.fc39                       @fedora-modular                                                
fribidi-devel.x86_64                                 1.0.13-2.fc39                       @fedora-modular                                                
gflags-devel.x86_64                                  2.2.2-12.fc39                       @fedora                                                        
glib2-devel.x86_64                                   2.78.3-1.fc39                       @updates                                                       
glibc-devel.i686                                     2.38-16.fc39                        @updates                                                       
glibc-devel.x86_64                                   2.38-16.fc39                        @updates                                                       
gmp-devel.x86_64                                     1:6.2.1-5.fc39                      @fedora-modular                                                
graphite2-devel.x86_64                               1.3.14-12.fc39                      @fedora-modular                                                
harfbuzz-devel.x86_64                                8.2.1-2.fc39                        @fedora-modular                                                
isl-devel.x86_64                                     0.16.1-18.fc39                      @fedora-modular                                                
java-17-openjdk-devel.x86_64                         1:17.0.9.0.9-3.fc39                 @updates                                                       
kernel-devel.x86_64                                  6.7.5-200.fc39                      @updates                                                       
kernel-devel.x86_64                                  6.7.6-200.fc39                      @updates                                                       
kernel-devel.x86_64                                  6.7.7-200.fc39                      @updates                                                       
kernel-devel-matched.x86_64                          6.7.7-200.fc39                      @updates                                                       
libICE-devel.x86_64                                  1.0.10-11.fc39                      @fedora-modular                                                
libRmath-devel.x86_64                                4.3.2-1.fc39                        @updates                                                       
libSM-devel.x86_64                                   1.2.3-13.fc39                       @fedora-modular                                                
libX11-devel.x86_64                                  1.8.7-1.fc39                        @fedora-modular                                                
libXau-devel.x86_64                                  1.0.11-3.fc39                       @fedora-modular                                                
libXext-devel.x86_64                                 1.3.5-3.fc39                        @fedora-modular                                                
libXfixes-devel.x86_64                               6.0.0-6.fc39                        @fedora-modular                                                
libXft-devel.x86_64                                  2.3.8-3.fc39                        @fedora-modular                                                
libXi-devel.x86_64                                   1.8.1-2.fc39                        @fedora-modular                                                
libXmu-devel.x86_64                                  1.1.4-3.fc39                        @fedora-modular                                                
libXrender-devel.x86_64                              0.9.11-3.fc39                       @fedora-modular                                                
libXt-devel.x86_64                                   1.2.1-5.fc39                        @fedora-modular                                                
libblkid-devel.x86_64                                2.39.3-6.fc39                       @updates                                                       
libdatrie-devel.x86_64                               0.2.13-7.fc39                       @fedora-modular                                                
libdrm-devel.x86_64                                  2.4.120-1.fc39                      @updates                                                       
libffi-devel.x86_64                                  3.4.4-4.fc39                        @fedora-modular                                                
libglvnd-core-devel.x86_64                           1:1.7.0-1.fc39                      @fedora-modular                                                
libglvnd-devel.x86_64                                1:1.7.0-1.fc39                      @fedora-modular                                                
libgnat-devel.x86_64                                 13.2.1-6.fc39                       @updates                                                       
libicu-devel.x86_64                                  73.2-2.fc39                         @fedora-modular                                                
libmount-devel.x86_64                                2.39.3-6.fc39                       @updates                                                       
libmpc-devel.x86_64                                  1.3.1-3.fc39                        @fedora-modular                                                
libpciaccess-devel.x86_64                            0.16-9.fc39                         @fedora-modular                                                
libpng-devel.x86_64                                  2:1.6.37-15.fc39                    @fedora-modular                                                
libquadmath-devel.x86_64                             13.2.1-6.fc39                       @updates                                                       
libselinux-devel.x86_64                              3.5-5.fc39                          @fedora-modular                                                
libsepol-devel.x86_64                                3.5-2.fc39                          @fedora-modular                                                
libstdc++-devel.x86_64                               13.2.1-6.fc39                       @updates                                                       
libthai-devel.x86_64                                 0.1.29-6.fc39                       @fedora-modular                                                
libtirpc-devel.x86_64                                1.3.4-0.rc2.fc39                    @updates                                                       
libxcb-devel.x86_64                                  1.13.1-12.fc39                      @fedora-modular                                                
libxcrypt-devel.i686                                 4.4.36-2.fc39                       @fedora-modular                                                
libxcrypt-devel.x86_64                               4.4.36-2.fc39                       @fedora-modular                                                
libxml2-devel.x86_64                                 2.10.4-3.fc39                       @fedora-modular                                                
libzstd-devel.x86_64                                 1.5.5-4.fc39                        @fedora-modular                                                
mesa-libGL-devel.x86_64                              23.3.6-1.fc39                       @updates                                                       
mesa-libGLU-devel.x86_64                             9.0.3-1.fc39                        @fedora-modular                                                
mpfr-devel.x86_64                                    4.2.0-3.fc39                        @fedora-modular                                                
ncurses-devel.x86_64                                 6.4-7.20230520.fc39.1               @updates                                                       
openssl-devel.x86_64                                 1:3.1.1-4.fc39                      @fedora-modular                                                
pango-devel.x86_64                                   1.51.0-1.fc39                       @fedora-modular                                                
pcre2-devel.x86_64                                   10.42-1.fc39.2                      @fedora-modular                                                
perl-Encode-devel.x86_64                             4:3.19-500.fc39                     @fedora                                                        
perl-devel.x86_64                                    4:5.38.2-502.fc39                   @updates                                                       
pixman-devel.x86_64                                  0.42.2-2.fc39                       @fedora-modular                                                
python3-devel.x86_64                                 3.12.2-2.fc39                       @updates                                                       
readline-devel.x86_64                                8.2-6.fc39                          @updates                                                       
sqlite-devel.x86_64                                  3.42.0-7.fc39                       @fedora-modular                                                
sysprof-capture-devel.x86_64                         45.1-1.fc39                         @updates                                                       
systemtap-devel.x86_64                               5.1~pre17062192g5fd8daba-1.fc39     @updates                                                       
systemtap-sdt-devel.x86_64                           5.1~pre17062192g5fd8daba-1.fc39     @updates                                                       
tcl-devel.x86_64                                     1:8.6.12-5.fc39                     @fedora-modular                                                
tk-devel.x86_64                                      1:8.6.12-5.fc39                     @fedora-modular                                                
tre-devel.x86_64                                     0.8.0-41.20140228gitc2f5d13.fc39    @fedora-modular                                                
wayland-protocols-devel.noarch                       1.33-1.fc39                         @updates                                                       
xorg-x11-proto-devel.noarch                          2023.2-2.fc39                       @fedora-modular                                                
xz-devel.x86_64                                      5.4.4-1.fc39                        @fedora-modular                                                
zlib-devel.x86_64                                    1.2.13-4.fc39                       @fedora-modular       
```
```

**Comment by Sharktheone** [comment]
Basically the same as #855, I've posted instructions on how to get it working.
TL;DR: use [my script](https://github.com/Sharktheone/arch-mojo) to install the libraries to a custom directory

> **Note**
> Note that I've tested it on F38 and there might be some issues with F39 that I need to address: https://github.com/modularml/mojo/issues/1816#issuecomment-1966933838

**Comment by smuniraj2018** [comment]
Closing this due to duplicate issues. Officially, we dont support Fedora yet. Please stay tuned for the future updates.
We are also aware some of the community users have managed to get it working. Thanks @Sharktheone.
